ucode.c

```diff
@@ -1,29 +1,30 @@
-#include "urun.h"
+#include "ucrun.h"
 
 static uc_parse_config_t config = {
-        .strict_declarations = false,
+        .strict_declarations = true,
         .lstrip_blocks = true,
```

Comment:
We should remove the trim_blocks and rstrip_blocks members as they are not applicable to raw mode